### ISSUE TITLE
Translation Context per Resource

### ISSUE THREAD

**Issue by fab1an** [marmelab/admin-on-rest#944]
Is there a way to customizes message per Resource. 
I want to change `aor.notification.created` but only for specific resources.

**Comment by djhi** [comment]
Hi, and thanks for your question. As explained in the [admin-on-rest contributing guide](https://github.com/marmelab/admin-on-rest/blob/master/CONTRIBUTING.md), the right place to ask a "How To" question, get usage advice, or troubleshoot your own code, is [StackOverFlow](http://stackoverflow.com/questions/tagged/admin-on-rest).

This makes your question easy to find by the core team, and the developer community. Unlike Github, StackOverFlow has great SEO, gamification, voting, and reputation. That's why we chose it, and decided to keep GitHub issues only for bugs and feature requests.

So I'm closing this issue, and inviting you to ask your question at:

http://stackoverflow.com/questions/tagged/admin-on-rest

And once you get a response, please continue to hang out on the admin-on-rest channel in StackOverflow. That way, you can help newcomers and share your expertise!

**Comment by fab1an** [comment]
Why did you close this?

There is nothing in the documentation about resource-specific translation.

If it's possible but not in the documenation, the documenation lacks.
If it's not possible, this is a feature-request ;).

**Comment by fab1an** [comment]
Forgive my perseverance, but in my humble opinion SO is NOT the right place to ask the question, because I don't need an idea on how to hack admin-on-rest so that it translates the messages, I need to know whether it's officially possible and just omitted in the documentation.

Should I rephrase this as Feature-Request and open a new issue? 

Thank your for your time.

**Comment by djhi** [comment]
No, it's not currently possible and I don't see how to make it happen.

The only thing we might do is to pass the resource's name to the translation mechanism so that you could include it in your message. Something like: `"aor.notification.created": "You created a %{resource}."`. If that's what you need, I'll reopen this issue as a feature request. If not, consider this won't fix anytime soon.

However, it's OSS, if you do find a way to implement this, please open a PR :) FYI, this is handled in the [crudResponse saga](https://github.com/marmelab/admin-on-rest/blob/master/src/sideEffect/saga/crudResponse.js#L30)

**Comment by fab1an** [comment]
How about overlaying all message per resource?
So that `{resource}.aor.notification.created` would get checked first, and after that `aor.notification.created`

**Comment by djhi** [comment]
Give it a try ;)
